src/Servers/Kestrel/Core/test/PipelineExtensionTests.cs

```diff
@@ -148,8 +148,8 @@ public void WriteAscii()
         }
 
         [Theory]
-        [InlineData(2, 1)]
-        [InlineData(3, 1)]
+        [InlineData(2, 2)]
+        [InlineData(3, 2)]
```

Comment:
We're now using GetMaxByteCount(1) to ensure we have enough space to write at least one character into the buffer, but for ASCII this returns 2 for some reason. I don't know how one char could become two ASCII bytes.